### ISSUE TITLE
edit window runs away from mouse cursor

### ISSUE THREAD

**Issue by blacksqr** [annotorious/annotorious-v1#81]
This behavior is seen with the example.html file included with the okfn plugin download:

The first time I click&drag to create an annotation, the edit window appears as expected.  I can drag the window around and it tracks the mouse cursor.

If I hit the cancel button on the edit window button and create a new annotation, another edit window pops up, but when I try to drag it, it shoots ahead of the mouse cursor, traveling about twice as far as the cursor in the drag direction.

Every time I repeat this and create a new edit window, the window travel distance increases relative to the mouse cursor movement, until the window is shooting across the screen with every small mouse movement.

The window also sometimes shoots back to its start position after I release the mouse button.

It looks like some value is not being re-initialized and is being incremented with each window creation.

I used plovr to generate the okfn plugin file for the master branch and verified that the problem still exists there.

I'm using Firefox v 24 on Ubuntu 12.04.

**Comment by rsimon** [comment]
Hi! Weird - I couldn't reproduce the problem (Chrome & Firefox both on Windows and Linux). Which version of Annotator have you been using it with? (I tested with v1.2.5.)

**Comment by blacksqr** [comment]
I used the example.html web page included with the okfn plugin version 0.3 download without changes.  That page includes the following link in the head section: <script src='http://assets.annotateit.org/annotator/v1.2.5/annotator-full.min.js'>

**Comment by rsimon** [comment]
Hm - very weird. I still can't reproduce this bug. I used the download package as you said, with no modifications. I click and drag, release the mouse. The OKFN editor pops up at the right place. I click 'Cancel'. I drag again & everything works fine. I release again; the OKFN editor again pops up at the right place.

Is this the exact sequence of what you did?

**Comment by blacksqr** [comment]
Yes, you seem to be doing everything right.  Thanks for taking the time, maybe more work with the package will lead me to further insight into what's going on.
